pages/requests/new/[ware].js

```diff
@@ -80,12 +82,14 @@ const NewRequest = () => {
   }
 
   const handleSubmit = async (event) => {
+    setButtonDisabled(true)
     if (!event.formData) {
       // these steps are needed for requests without a dynamic form
       // but error on the event resulting from the react json form
       event.preventDefault()
       event.stopPropagation()
       setValidated(true)
+      setFormSubmitting(true)
```

Comment:
I know you said that `setFormSubmitting` needed to be further down, but I think it should be after the `if` block. as is, the value would only get set for requests without a dynamic form.

Reply:
moved it down and tested it with a dynamic form and it still works as expected 👍